README.md

```diff
@@ -41,16 +43,22 @@ Long term developments of this project will seek to re-deploy more complex ML pa
 This folder contains the fortran neural net extracted from the [code referenced above](https://github.com/yaniyuval/Neural_nework_parameterization/tree/v.1.0.3), along with any dependencies, that may be compiled as a standalone fortran module.
 
 Currently there is code that can be built on CSD3 using the included shell script.
-
 This now needs cleaning up, testing, and a proper makefile creating (see open issues #9 and #10).
 
 ### ``torch_nets/``
 The directory contains the PyTorch versions of the neural networks we are interested in.
 
+### ``CAM_interface/``
+The directory contains the additional files or details to interface the code with the CAM atmospheric model
+as part of the CESM model suite. It also includes a link to an implementation in a fork of CAM.
+
 
 ## Contributing
 
-This repository is currently private as it is new and work in progress.
+Contributions to the repository are welcome, particularly from anyone seeking to implement the
+parameterisation in another atmospheric model.
+We welcome code and details for interfacing.
```

Comment:
What do you mean by "details for interfacing"? Interface specifications?

Reply:
I have clarified in https://github.com/m2lines/convection-parameterization-in-CAM/pull/44/commits/033cb038fdc514b7c5d1c8f8a0577a52dc3f8f90